website/docs/docs/build/incremental-microbatch.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "About microbatch incremental models"
+description: "Learn about the 'microbatch' strategy for incremental models."
+id: "incremental-microbatch"
+---
+
+# About microbatch incremental models <Lifecycle status="beta" />
+
+:::info Microbatch
+
+The `microbatch` strategy is available in [dbt Cloud Versionless](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless) and dbt Core v1.9.
+
```

Comment:
do we mean to set it to true in their dbt cloud environments, yes?

Reply:
I think we can just say "to true" and end there